src/main/java/org/seqdoop/hadoop_bam/util/BGZFSplitGuesser.java

```diff
@@ -23,25 +23,37 @@
 package org.seqdoop.hadoop_bam.util;
 
 import java.io.IOException;
+import java.io.InputStream;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.util.Arrays;
 
 import htsjdk.samtools.util.BlockCompressedInputStream;
 
 import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.Seekable;
 
 public class BGZFSplitGuesser {
```

Comment:
The duplication is already present, and is not trivial to remove as it has a slightly different signature and method body. See https://github.com/HadoopGenomics/Hadoop-BAM/pull/77 where I removed some duplication. I'd like to tackle the remaining duplication in a separate issue.